compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -132,10 +137,11 @@ struct ConvertToROCDLPass : public ConvertToROCDLBase<ConvertToROCDLPass> {
     // Run Vector -> Vector transformations ahead of conversion to LLVM.
     {
       RewritePatternSet patterns(&getContext());
+      arith::populateArithToAMDGPUConversionPatterns(patterns);
```

Comment:
Can you add a comment describing which subset of arith ops this converts? I'd assume that only these ops that don't have a dedicate/efficient lowering through the llvm dialect?